src/containers/CheckoutAuthPage.jsx

```diff
@@ -141,7 +141,7 @@ class CheckoutAuthPage extends React.Component {
                   <p>
                     {intl.get('create-an-account-message')}
                   </p>
-                  <button className="btn btn-primary checkout-auth-option-register-btn" type="button" onClick={() => { history.push('/registration', { returnPage: '/checkout' }); }}>
+                  <button className="vestri-btn primary wide" type="button" onClick={() => { history.push('/registration', { returnPage: '/checkout' }); }}>
```

Comment:
The independent class names (example: checkout-auth-option-register-btn) are meant more for readability and for automation. I'd prefer to keep some or add IDs where applicable rather than having generic elements in some places.